web/src/beta/components/fields/CameraField/index.tsx

```diff
@@ -0,0 +1,313 @@
+import { useCallback, useState } from "react";
+
+import Button from "@reearth/beta/components/Button";
+import NumberInput from "@reearth/beta/components/fields/common/NumberInput";
+import Icon from "@reearth/beta/components/Icon";
+import * as Popover from "@reearth/beta/components/Popover";
+import Slider from "@reearth/beta/components/Slider";
+import Text from "@reearth/beta/components/Text";
+import { useT } from "@reearth/services/i18n";
+import { styled, useTheme } from "@reearth/services/theme";
+
+import Property from "..";
+
+type CameraInput = {
+  name: string;
+  field: keyof CameraValue;
+};
+// Constants
+const CAMERA_XYZ: CameraInput[] = [
+  {
+    name: "Latitude",
+    field: "lat",
+  },
+  {
+    name: "Longitude",
+    field: "lng",
+  },
+  {
+    name: "Height",
+    field: "height",
+  },
+];
+
+const CAMERA_ANGLE: CameraInput[] = [
+  {
+    name: "Heading",
+    field: "heading",
+  },
+  {
+    name: "Pitch",
+    field: "pitch",
+  },
+  {
+    name: "Roll",
+    field: "roll",
+  },
+];
+
+export type CameraValue = {
+  lat: number;
+  lng: number;
+  height: number;
+  heading: number;
+  pitch: number;
+  roll: number;
+  fov: number;
+};
+
+export type Props = {
+  name?: string;
+  description?: string;
+  value?: CameraValue;
+  disabled?: boolean;
+  onCapture?: () => void;
+  onJump?: (input: CameraValue) => void;
+  onClean?: () => void;
+  onChange: (value: CameraValue) => void;
+};
+
+// Component
```

Comment:
Don't need obvious comments like this. Typically only do comments if something isn't obvious, and before writing comments think of how you can rename, refactor the function, etc to make it more understandable without comments first 🙏🏻

Reply:
Done